StripeFinancialConnections/StripeFinancialConnections/Source/Native/LinkAccountPicker/LinkAccountPickerViewController.swift

```diff
@@ -116,7 +117,15 @@ final class LinkAccountPickerViewController: UIViewController {
                 guard let self = self else { return }
                 switch result {
                 case .success(let networkedAccountsResponse):
-                    if let returningNetworkingUserAccountPicker = networkedAccountsResponse.display?.text?.returningNetworkingUserAccountPicker {
+                    if networkedAccountsResponse.data.isEmpty {
+                        let nextPaneOnAddAccount = dataSource.nextPaneOnAddAccount ?? .institutionPicker
```

Comment:
This is a value we get back when fetching networked accounts. It's [technically optional](https://stripe.sourcegraphcloud.com/git.corp.stripe.com/stripe-internal/pay-server/-/blob/lib/bank_connections/api/networking/resources/networked_accounts_list_resource.rb?L37-38), but seems to [always return a value](https://stripe.sourcegraphcloud.com/git.corp.stripe.com/stripe-internal/pay-server/-/blob/lib/bank_connections/api/networking/methods/link_account_session_networked_accounts_retrieve_method.rb?L177-196).